packages/core/src/browser/style/hover-service.css

```diff
@@ -27,7 +27,7 @@
   background-color: var(--theia-editorHoverWidget-background);
   border: 1px solid var(--theia-editorHoverWidget-border);
   padding: var(--theia-ui-padding);
-  max-width: var(--theia-hover-max-width);
+  width: var(--theia-hover-width);
```

Comment:
This change actually breaks other hovers we have in the framework (ex: statusbar).
Can we instead only style the enhanced preview with the proper width and not update the more generic `theia-hover`:


https://github.com/eclipse-theia/theia/assets/40359487/bd82c119-3285-426f-b061-43ca94dd37c3

Reply:
Good catch! I reworked the last commit, so that both variables are there, but the `--theia-hover-preview-width` variable is only used by the preview hovers.